README.md

```diff
@@ -55,230 +57,82 @@ Log4j
 
-cfClient.update(Message message)
+```java
+public class SimpleExample {
+    public static void main(String[] args) {
+        try {
+            /**
+             * Put the API Key here from your environment
+             * Initialize the CfClient
+             */
+            String apiKey = "<your api key here>";
+            CfClient cfClient = new CfClient(apiKey, Config.builder().build());
+            cfClient.waitForInitialization();
+    
+            while(true) {
+                /**
+                 * Sleep for sometime before printing the value of the flag
+                 */
+                Thread.sleep(2000);
+                /**
+                 * This is a sample boolean flag. You can replace the flag value with
+                 * the identifier of your feature flag
+                 */
+                boolean result = cfClient.boolVariation("<your feature flag id here>", target, <default value>);
+                log.info("Boolean variation is " + result);
+            }
+        } catch(Exception e) {
+            e.printStackTrace();
+        }
+    }
+}
 ```
 
-## Shutting down the SDK
-
-To avoid potential memory leak, when SDK is no longer needed
-(when the app is closed, for example), a caller should call this method:
 
-```
-cfClient.close();
-```
+### Additional Reading
 
-## Logger configuration
+Further examples and config options are in the further reading section:
 
-SDK uses Sl4j as facade to any MDC supported logging library like logback, log4j2 and others.
-You probably already have log4j or logback in your project, 
-and we provide variables for your log layout file.
+[Further Reading](docs/further_reading.md)
 
-Supported variables:
-```
-variables     description                                           scope
-SDK         - sdk language                                          system
-version     - version of SDK                                        system
-flag        - current flag evaluation                               context
-target      - target used for evaluation                            context
-requestId   - request identifer for interacting with FF backend     context
-```
 
-sample logback file
-```xml
-<?xml version="1.0" encoding="UTF-8"?>
-
-<configuration debug="false" scan="true">
-
-    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder>
-            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36}.%M\(%line\) - SDK=property{SDK}, Version=property{version}, flag=%X{flag}, target=%X{target}, requestID=%X{requestId} - %msg%n
-            </pattern>
-        </encoder>
-    </appender>
-    <root level="INFO">
-        <appender-ref ref="STDOUT" />
-    </root>
-</configuration>
-```
+-------------------------
+[Harness](https://www.harness.io/) is a feature management platform that helps teams to build better software and to
+test features quicker.
```

Comment:
This layout formats are just showing how to use system and context vars:
* property{SDK}
* property{version}
* %X{flag}
* %X{target}

Reply:
I will re-add the log4j example in `Further Reading`